linux/opt/sage/bin/make_env_vars_file.sh

```diff
@@ -32,8 +32,10 @@ extract_tag_value() {
 DEPARTMENT=$(extract_tag_value Department)
 PROJECT=$(extract_tag_value Project)
 PROVISIONING_PRINCIPAL_ARN=$(extract_tag_value 'aws:servicecatalog:provisioningPrincipalArn')
-ACCESS_APPROVED_ROLEID=$(extract_tag_value AccessApprovedRoleId)
 PRINCIPAL_ID=${PROVISIONING_PRINCIPAL_ARN##*/}  #Immutable Synapse userid derived from assume-role session name
+ASSUMED_ROLE_NAME=$(cut -d'/' -f2 <<< ${PROVISIONING_PRINCIPAL_ARN}) # the SC end user assumed role name
+ACCESS_APPROVED_ROLEID=$(/usr/bin/aws --region $AWS_REGION \     # the SC end user assumed role ID
+  iam get-role --query Role.RoleId --out text --role-name ${ASSUMED_ROLE_NAME})
```

Comment:
does the instance profile on provisioned instances have access to do this?

Reply:
yes, it does.  right here: https://github.com/Sage-Bionetworks/service-catalog-library/blob/master/ec2/sc-ec2-linux-jumpcloud-v1.0.1.yaml#L206